webapp/src/types/gitlab_items.ts

```diff
@@ -0,0 +1,64 @@
+import {Theme} from 'mattermost-redux/types/preferences';
+import * as CSS from 'csstype';
+import { notificationReasons } from 'src/components/sidebar_right/gitlab_items';
+
+export interface Label {
+    id: number;
+    name: string;
+    color: CSS.Properties;
+    text_color: CSS.Properties;
```

Comment:
I think defacto standard is camel case  like `textColor` unless there is a valid reason for not doing it

Reply:
oh, never mind, I see the reason to match the json structure